### ISSUE TITLE
WCAG 2.2 Violation - There are controls with non-descriptive accessible names present within the interface.

### ISSUE THREAD

**Issue by ivrouwe-ubc** [ubc/wp-comments-chat-ui#11]
#### This violates the following WCAG (Web Content Accessibility Guidelines) 2.2 Success Criteria:

- 

### User Impact

Some users - such as people using voice control software - may be unable to access, focus, or activate the controls. Screen reader users may be confused by the controls being announced by a different accessible name than their visible labels.

#### Further information:

1. 

### Instances

1. #49 
2. #50
